examples/plot_clickable_image.py

```diff
@@ -0,0 +1,70 @@
+"""
+=========================================
+Demonstration of how to use ClickableImage / generate_2d_layout.
+=========================================
+
+In this example, we open an image file, then use ClickableImage to
+return 2D locations of mouse clicks (or load a file already created).
+Then, we use generate_2d_layout to turn those xy positions into a layout
+for use with plotting topo maps. In this way, you can take arbitrary xy
+positions and turn them into a plottable layout.
+"""
+# Authors: Christopher Holdgraf <choldgraf@berkeley.edu>
+#
+# License: BSD (3-clause)
+import sys
+sys.path.insert(0, '../')
```

Comment:
with `python setup.py develop` you never have to worry about re-building since we are pure python, doing `git checkout` automatically switches the active version